src/scss/components/optional/_content-header.scss

```diff
@@ -1,3 +1,7 @@
+@use '../../../../../pie-design-tokens/dist/jet' as dt;
```

Comment:
Do these absolute references also need to be updated to point at the npm package?